makewine.sh

```diff
@@ -18,6 +18,7 @@
 
 export PREFIX="${PREFIX:-${HOME}/usr}"
 export WINE_VERSION="${WINE_VERSION:-wine-1.9.16}"
+[[ -z "${THREADS}" ]] && export THREADS="$(awk '{ threads+=($0 ~ "^processor") }END{ print threads+1 }' /proc/cpuinfo)"
```

Comment:
I wonder how often the `nproc` command is pre-installed on Linux systems.  It's typically available on Ubuntu by default.  Also, interesting approach with awk.  Another awk way is (not that you should change it to my way):

```bash
awk '/^processor/ { threads++ }END{ print threads+1 }' /proc/cpuinfo
```